### DISCUSSION TITLE
Calculating convective heat transfer with BOS

### DISCUSSION THREAD

**Discussion by Sadeghi6872** [firemodels/fds#13310] [Q&A]
Hi All, 
Hope you are doing well. I have Background Oriented Schlieren images of the fire which give me the displacement field and want to calculate the convective heat flux. I wonder how I can go for this or if I can FDS. 

I would appreciate any idea. 

Best,
Mohamad

**Comment by Sadeghi6872** [comment]
Hi Again, 

I am wondering if there is any way in FDS to have a 2-D contour of convective heat flux in a defined domain at the centerline of the test bed with Excelsior particles. 

Thanks.

**Reply by mcgratta** [reply]
You need to be more specific. Do you want the convective heat flux to the ground? If so, you can use a boundary (BNDF) output for `CONVECTIVE HEAT FLUX`.

**Comment by Sadeghi6872** [comment]
Hi Kevin, 

Thank you very much for the response. I need convective heat flux of the flame and plume region as a 2-D contour.

**Reply by mcgratta** [reply]
I still don't know exactly what you mean, but I will guess. Search for `ENTHALPY FLUX Z` in the User's Guide.

**Comment by Sadeghi6872** [comment]
Dear Kevin,

I apologize for any inconvenience and appreciate your feedback. I have added ENTHALPY FLUX to my FDS code with the following line:

    &DEVC ID='q-dot', XB=-1.8,0,0.5,0.5,0.1,1, QUANTITY='ENTHALPY FLUX Y', SPATIAL_STATISTIC='AREA INTEGRAL' /

This device produces a single data point for each time step, as seen in my _devc.csv file.

I currently use the following slice file to display a temperature contour on the midplane (y = 0.5):

    &SLCF PBY=0.5, QUANTITY='TEMPERATURE', VECTOR=T /

However, I now need a similar slice configuration to show the convective heat flux, and I am unsure how to proceed. I have attached my FDS code for your reference.

Thank you very much for your assistance.

Best regards,
Mohamad


[Wind100.txt](https://github.com/user-attachments/files/19677215/Wind100.txt)

**Reply by Vojta-Salek** [reply]
Hello, `&DEVC` always produces a single data output. Although the `XB` input is a plane, you call `SPATIAL_STATISTIC='AREA INTEGRAL'` and the results of the integral is a scalar (total heat transfer over the area).

If you need exactly the same slice file, why don't you analogically use `&SLCF PBY=0.5, QUANTITY='ENTHALPY FLUX Z', VECTOR=T /`?

You have a large and relatively complex simulation. I strongly recommend reading the User's Guide carefully, as it is always the user's responsibility to understand what FDS does.
